tests/inheritance/test_join_table_inheritance.py

```diff
@@ -119,6 +122,27 @@ def test_primary_keys(self):
         assert 'id' in table.primary_key.columns
         assert tx_column in table.primary_key.columns
 
+    def test_versions(self):
```

Comment:
Probably easier to merge this in if you mark this test as `xfail`, i.e. a known issue for now. This way we can merge it in despite not having a fix prepared without breaking the build in the meantime.

Simply add the decorator `@pytest.mark.xfail`

[From the docs](https://docs.pytest.org/en/2.9.0/skipping.html#skip-and-xfail-dealing-with-tests-that-can-not-succeed)
> And xfail means that your test can run but you expect it to fail because there is an implementation problem.